src/components/compress/index.tsx

```diff
@@ -328,7 +341,9 @@ export default class Compress extends Component<Props, State> {
         });
       }
 
-      this.setState(newState);
+      this.setState(newState, () => {
+        this.updateDocumentTitle(file.name);
```

Comment:
I don't think we need to use the callback here, right? We can just call `updateDocumentTitle`.

Reply:
I thought we might want to display filename only after the file is loaded into the state, but if there's no reason - I'll change it.

Reply:
You'll always see the result of `setState` before the next paint (unless I'm missing something), so without the callback is fine IMO.